css/main.css

```diff
@@ -167,11 +170,15 @@ kbd {
 
 .contentContainer{
 	display: flex;
-	height: 100%;
+	min-width: 0;
+	min-height: 0;
```

Comment:
I don't think these mins are doing anything here since we will always have content in the container and have flex set to 1. So it'll fill the rest.

Reply:
Believe it or not, these two lines are actually the most important in the entire flexbox overhaul lol. Apparently, flex by default makes containers at least the size required to show their content, even if flex: 1 is set. So overriding these defaults with min width and height is basically saying the flex boxes can get as small as they need to be (in this case, the flex 1 size needed to fill remaining space below the top bar) with the rest overflowing. Before these lines were added, the left panel was actually astronomically long to fit guide content so you couldn't scroll properly through them and they weren't actually filling the window as intended.